zone/client.cpp

```diff
@@ -6912,6 +6912,34 @@ bool Client::IsMarried()
 	return m_epp.married_character_id != 0;
 }
 
+void Client::SetCharExportFlag(uint8 flag)
+{
+	if (flag == 0) {
+		m_epp.char_export_flag = 0;
+		Save(1);
+		Message(CC_Default, "Character export disabled.");
+		return;
+	}
+	if (flag == 1) {
```

Comment:
Change this to else if for optimization purposes.

Reply:
Updated to if-else if chain